project/ProjectPlugin.scala

```diff
@@ -0,0 +1,114 @@
+      Map(
+        'evaluator     -> "0.2.0-SNAPSHOT",
+        'stdlib        -> "0.4.0",
+        'cats          -> "0.4.0",
+        'shapeless     -> "0.4.0",
+        'doobie        -> "0.4.0",
+        'scalacheck    -> "0.4.0",
+        'scalatutorial -> "0.4.0",
+        'fpinscala     -> "0.4.0"
+      ) ++ Map(
+        // JVM Versions
+        'bootstrap      -> "3.3.7",
+        'classutil      -> "1.1.2",
+        'commonsio      -> "2.5",
+        'freestyle      -> "0.1.0-SNAPSHOT",
+        'highlightjs    -> "9.2.0",
+        'knockoff       -> "0.8.3",
+        'newrelic       -> "3.29.0",
+        'postgres       -> "9.3-1102-jdbc41",
+        'scalajsscripts -> "1.0.0",
+        'scalariform    -> "0.1.8",
+        'upickle        -> "0.4.3",
+        'webjars        -> "2.5.0-4"
+      ) ++ Map(
+        // JS Versions
+        'jquery        -> "2.1.3",
+        'scalajsdom    -> "0.9.1",
+        'scalajsjquery -> "0.9.1",
+        'scalatags     -> "0.6.3",
+        'utest         -> "0.4.5"
+      )
+
+    implicit class Exclude(module: ModuleID) {
+
+      def x(org: String, artifact: String): ModuleID = module.exclude(org, artifact)
+
+      def xscalaz: ModuleID = x("org.scalaz", "scalaz-concurrent")
+
+      def xdoobie: ModuleID = module.excludeAll(ExclusionRule("org.tpolecat"))
+
+      def xscalajs: ModuleID =
+        x("org.scala-js", "scalajs-library") x ("org.scala-js", "scalajs-dom")
+    }
+
+  }
+
+  override def projectSettings: Seq[Def.Setting[_]] =
+    Seq(
+      description := "Scala Exercises: The path to enlightenment",
+      startYear := Option(2016),
+      orgGithubSetting := GitHubSettings(
+        organization = "scala-exercises",
+        project = name.value,
+        organizationName = "Scala Exercises",
+        groupId = "org.scala-exercises",
+        organizationHomePage = url("https://www.scala-exercises.org"),
+        organizationEmail = "hello@47deg.com"
+      ),
+      orgLicenseSetting := ApacheLicense,
+      scalaVersion := "2.11.8",
+      scalaOrganization := "org.scala-lang",
+      crossScalaVersions := Seq("2.11.8"),
+      resolvers ++= Seq(
+        Resolver.mavenLocal,
+        Resolver.sonatypeRepo("snapshots"),
+        Resolver.sonatypeRepo("releases"),
+        "Typesafe Releases" at "http://repo.typesafe.com/typesafe/releases/",
+        "Typesafe Maven Releases" at "http://repo.typesafe.com/typesafe/maven-releases/",
+        "scalaz-bintray" at "http://dl.bintray.com/scalaz/releases",
+        Resolver.url("sbt-plugins", url("https://dl.bintray.com/ssidorenko/sbt-plugins/"))(
+          Resolver.ivyStylePatterns)
+      ),
+      scalacOptions += "-Xplugin-require:macroparadise",
+      javacOptions ++= Seq("-encoding", "UTF-8", "-Xlint:-options"),
+      fork in Test := false,
+      parallelExecution in Test := false,
+      cancelable in Global := true,
+      headers := Map(
+        "scala" -> (HeaderPattern.cStyleBlockComment,
+        s"""|/*
```

Comment:
It would be nice if `org-policies` was able to generate the copyright headers automatically from the org name, year and licenses selected and optionally allow the user to override it with a custom header such as this one. Not saying we change that now, but that would be a great feature.